items/README.md

```diff
@@ -0,0 +1,19 @@
+# Items
+Characters can collect an assortment of many items in their journeys. Here we list some examples, but the extent of what items can exist and how they are used is is for the players and DM to discover.
+
+## General Template
```

Comment:
Very object oriented.

I agree with the principal, but I think you need to throw away the reference to "general template".
Just define what an item is and has. Weapons should specify that a weapon is an item.